catalog/core/catalog-core-metacardgroomerplugin/src/main/java/ddf/catalog/plugin/groomer/metacard/StandardMetacardGroomerPlugin.java

```diff
@@ -72,6 +73,15 @@ protected void applyCreatedOperationRules(CreateRequest createRequest, Metacard
             aMetacard.setAttribute(new AttributeImpl(Metacard.TAGS,
                     Collections.singletonList(Metacard.DEFAULT_TAG)));
         }
+
+        // upon creation of the metacard, generate attributes for created, modified dates
```

Comment:
Doing empty checks on `created`, but `modified` will always be updated on both a create and update operation.